hwdb/parse_hwdb.py

```diff
@@ -112,6 +112,8 @@ def property_grammar():
              ('ID_INPUT_TOUCHPAD_INTEGRATION', Or(('internal', 'external'))),
              ('XKB_FIXED_LAYOUT', STRING),
              ('XKB_FIXED_VARIANT', STRING),
+             ('KEYBOARD_LED_NUMLOCK', Literal('0')),
+             ('KEYBOARD_LED_CAPSLOCK', Literal('0')),
```

Comment:
Yes, it could be used for overrides. hwdb does all possible matches, and in order, and the latest property specified with a given key wins. But we can always add that later.